mautic/mautic
# Unique lead fields are now checked for unique on Lead form.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Issue(s) addressed                     | Fixes #10282 



#### Description:
Unique contact fields now have a server side validation.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a unique field (you already have an email) for contact, and for company.
3. Create a contact with email or one of the newly created fields in different combinations or standalone.
4. Create another contact with the same field(s) from step 2 in both Quick add and New/Edit page.
5. If you have possibility - test the same via API.
6. Repeat steps above after setting `company_unique_identifiers_operator` config value to `AND`
